### DISCUSSION TITLE
PDF Generate and Auto email to different customers

### DISCUSSION THREAD

**Discussion by anoop-mathew** [Budibase/budibase#6814] [Ideas]
If there any option to generate pdf over a specific template uploaded/ created with the content in the table. (For example if a client wish to generate an invoice/voucher/certificate and to send it to the corresponding customers email)

**Comment by melohagan** [comment]
Hey @anoop-mathew 

Currently there is no way to generate a PDF from table contents, but it's a good idea! 

It's a very popular idea: https://github.com/Budibase/budibase/issues/1372

**Reply by sanskritkm** [reply]
pls we need to export table content as PDF or print out
